app/views/users/show.html.erb

```diff
@@ -2,16 +2,17 @@
   <!-- pseudoflash because putting it in a real one was making for some ugly code -->
   <div class="flash notice" id="first-login-help-banner">
     <p>
-      Hi! It looks like you've just logged into the archive for the first time. 
-      <%= link_to_modal ts("Learn some tips and tricks"), :for => first_login_help_path, :title => "First login help" %> 
-      or dismiss this message permanently.
+      <%= t(".login_banner.welcome_text", help_link: link_to_modal(t(".login_banner.help_link"), :for => first_login_help_path, :title => t(".login_banner.help_title"))).html_safe %>
     </p>
-      <%= form_tag end_first_login_user_path(current_user), :method => :post, :remote => true do %>
+    <p>
+      <%= t(".login_banner.abuse_text", abuse_link: link_to(t(".login_banner.abuse_link"), "/abuse_reports/new")).html_safe %>
+    </p>
+    <%= form_tag end_first_login_user_path(current_user), :method => :post, :remote => true do %>
     <p class="submit actions">
-      <%= submit_tag ts("Dismiss permanently") %>
-      <%= link_to "x", nil, :remote => true, :id => "hide-first-login-help", :title => "Hide first login help banner" %>
+      <%= submit_tag t(".login_banner.dismiss") %>
+      <%= link_to "x", nil, :remote => true, :id => "hide-first-login-help", :title => t(".login_banner.hide") %>
```

Comment:
One more annoying "as long as you're here..." thing: can you make sure this is [the times symbol like we use in admin banners](https://github.com/otwcode/otwarchive/blob/f057b6b9f85ccdd26dcff2aca2efa17cf7f38e26/app/views/layouts/_banner.html.erb#L18) and not a letter `x`? (I _think_ it's currently the letter, but it's hard to tell in in certain fonts unless you see the two things side by side.)

Reply:
TIL "&times;" was a thing.  This was totally the letter.